openzwave-sys/build.rs

```diff
@@ -6,10 +66,28 @@ fn main() {
      .file("openzwave-c/manager.cc")
      .file("openzwave-c/notification.cc")
      .file("openzwave-c/value_classes/value_id.cc")
-     .include("/usr/include/openzwave") // workaround for https://github.com/OpenZWave/open-zwave/issues/771
-     .include("/usr/local/include/openzwave")
      .cpp(true)
-     .flag("-std=c++11") // to iterate with ranges
-     .compile("libopenzwave-c.a");
-    println!("cargo:rustc-flags=-l openzwave");
+     .flag("-std=c++11"); // to iterate with ranges
+
+    if find_include_path(&mut c).is_ok() {
```

Comment:
likely we'll also want to skip it if we're cross compiling -- but that's for a separate PR too.